tests/TestCase/Consumption/QueueExtensionTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org/)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org/)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         0.1.0
+ * @license       https://opensource.org/licenses/MIT MIT License
+ */
+namespace Queue\Test\TestCase\Consumption;
+
+use Cake\Event\EventList;
+use Cake\Queue\Consumption\QueueExtension;
+use Cake\TestSuite\TestCase;
+use Enqueue\Consumption\Context\PostMessageReceived;
+use Enqueue\Consumption\Result;
+use Enqueue\Null\NullConnectionFactory;
+use Enqueue\Null\NullMessage;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Psr\Log\Test\TestLogger;
+
+/**
+ * Class QueueExtensionTest
+ *
+ * @package Queue\Test\TestCase\Consumption
```

Comment:
Nitpick: package tags are unnecessary in names namespaced classes.

Reply:
it came from the copypaste desert, fixed